src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -686,6 +710,12 @@ export default class FilingHistoryList extends Vue {
         item.expiry = filing.data.restoration?.expiry
       }
 
+      // add properties for limited restoration extensions
+      if (EnumUtilities.isTypeRestorationExtension(filing)) {
+        item.isTypeLimitedRestorationExtension = true
+        item.expiry = filing.data.restoration?.expiry
+      }
+
```

Comment:
Would it be possible to add these attributes inside the `LimitedRestorationExtension` filing history component?

Reply:
Are you OK with leaving this until Phase 2 of this work (next PR)?